lib/split/dashboard/public/style.css

```diff
@@ -326,3 +329,8 @@ a.button.green:focus, button.green:focus, input[type="submit"].green:focus {
   display: inline-block;
   padding: 5px;
 }
+
+.divider {
+  display: inline-block;
+  margin-left: 10px;
+}
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/dashboard/public/style.css#L334-L337